app/common/services/data-export.service.js

```diff
@@ -67,6 +70,19 @@ function DataExport($rootScope, ExportJobEndpoint,  Notify, $window, $timeout, $
         }, CONST.EXPORT_POLLING_INTERVAL);
     }
 
+    function updateExportJobsList(job) {
```

Comment:
I think you need to process the list here as well. When I start a new job, the expiry-date shows up as 'invalid' if I don't reload the page

Reply:
good catch, thanks!

Reply:
ok so it ended up being a different problem involving the date of pre processed jobs (which was being converted again and resulting in an invalid date) . I also found a bug with the sorting of the jobs which is fixed now.